docs/index.asciidoc

```diff
@@ -118,6 +119,14 @@ To improve accuracy of the parsing plugin, users will have the option of sending
 
 To ensure that the memory usage of the plugin is not unbounded, a maximum size can be setup for the n-gram dictionaries used to parse logs. This value determines the maximum size for the single-gram, double-gram, and tri-gram hash-maps respectively.
 
+[id="plugins-{type}s-{plugin}-regexes"]
+===== `regexes`
+
+  * Value type is <<string,string>>
+  * Default value is `[]`
+
+The regexes variable allows regexes to be instered into the parsing pipeline to capture known dynamic tokens before the log is passed through the pipeline. Increasing the number of known dynamic tokens extracted in preprocessing will make the parsed results more accurate.
```

Comment:
nit: inserted, not instered.

Can you include examples of regexes that people could supply